using-mobile-extensions/mobile-core/lifecycle/lifecycle-api-reference.md

```diff
@@ -124,6 +124,8 @@ Pause Lifecycle data collection when your app has entered the background:
 {% tab title="React Native" %}
 ### JavaScript
 
+> Note: Implementing Lifecycle via Javascript may lead to inaccurate Lifecycle metrics, therefore we recommend implementing Lifecycle in native [Android and iOS code](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/lifecycle). However, if implementing Lifecycle in Javascript is required you can use the [`AppState`](https://facebook.github.io/react-native/docs/appstate) to receive notifications about when your app enters foreground/background. Based on the `AppState` you can make the corresponding calls to `lifecycleStart` and `lifecyclePause`.
```

Comment:
- Please change "Note" to "Tip" and make it bold.

- Please reword the text so that it reads as follows:

"Tip: Implementing Lifecycle via Javascript might lead to inaccurate Lifecycle metrics. As a result, we recommend that you implement Lifecycle in native [Android and iOS code](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/lifecycle). However, if you need to implement Lifecycle in Javascript, you can use  [`AppState`](https://facebook.github.io/react-native/docs/appstate) to receive notifications about when your app enters foreground/background. Based on the `AppState` you can make the corresponding calls to `lifecycleStart` and `lifecyclePause`."